### DISCUSSION TITLE
Not Able to Import Cables CSV

### DISCUSSION THREAD

**Discussion by mahmoudkhalil2** [netbox-community/netbox#11418] [Q&A] [answered]
Hello, I faced the below error when trying to import CSV file for the Cables tab.

how i can import cables using CSV file whithout facing this error, I used the exported CSV file from the Netbox Demo as a template.

and this is the csv file i used to import.

[netbox_cables(2).csv](https://github.com/netbox-community/netbox/files/10361381/netbox_cables.2.csv)

![image](https://user-images.githubusercontent.com/121564738/211048684-75e0e1bb-db91-4a0d-8720-e2c81b426a34.png)

**Comment by jeremystretch** [comment, marked as answer]
Looks like you're running into #11210. We're currently evaluating a fix for this.

**Reply by mahmoudkhalil2** [reply]
Thanks to update me after the fix.

**Reply by jeremystretch** [reply]
You can subscribe to the GitHub issue to receive notifications as progress is made.
